watertap/flowsheets/full_water_resource_recovery_facility/BSM2_P_extension.py

```diff
@@ -872,9 +872,82 @@ def display_performance_metrics(m):
         pyo.units.get_units(m.fs.AD.liquid_phase.properties_in[0].flow_vol),
     )
 
+    print("---- Feed Metrics----")
+    print(
+        "Feed TSS concentration",
+        pyo.value(m.fs.FeedWater.properties[0].TSS),
+        pyo.units.get_units(m.fs.FeedWater.properties[0].TSS),
+    )
+    print(
+        "Feed COD concentration",
+        pyo.value(m.fs.FeedWater.properties[0].COD),
+        pyo.units.get_units(m.fs.FeedWater.properties[0].COD),
+    )
+    print(
+        "BOD5 concentration",
+        pyo.value(m.fs.FeedWater.properties[0].BOD5["effluent"]),
```

Comment:
Should the index be influent instead of effluent?

Reply:
Yes - although technically it should be "raw". I honestly forgot I even printed the influent metrics already, but this is just something I added without putting too much thought in. I should verify that the equations for influent really are identical to the equations for effluents. We know, at the very least, that the equation for BOD5 differs slightly, which is why we have this config option implemented